### PR TITLE
MC test batch - applications protocol x509

### PR DESCRIPTION
**Pull request by omercier**
## Description

Part of the test batch

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [x] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

REFS: MON-20691

### CODE REVIEW

File: pp/integrations/plugin-packs/procedures/applications-protocol-x509.md [hunk 1]
```diff
@@ -2,59 +2,361 @@
...
+Here is the list of services for this connector, detailing all metrics linked to each service.
+
+<Tabs groupId="sync">
+<TabItem value="X509-Certificate" label="X509-Certificate">
+
+| Metric name | Unit  |
+|:------------|:------|
+| status      | N/A   |
+
+> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-x509.md:41]: is this necessary?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-x509.md [hunk 2]
```diff
@@ -2,59 +2,362 @@
...
+Voici le tableau des services pour ce connecteur, détaillant les métriques rattachées à chaque service.
+
+<Tabs groupId="sync">
+<TabItem value="X509-Certificate" label="X509-Certificate">
+
+| Métrique    | Unité |
+|:------------|:------|
+| status      | N/A   |
+
+> Pour obtenir ce nouveau format de métrique, incluez la valeur **--use-new-perfdata** dans la macro de service **EXTRAOPTIONS**.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-x509.md:42]: besoin de ça ici?

File: pp/integrations/plugin-packs/procedures/applications-protocol-x509.md [hunk 3]
```diff
@@ -2,59 +2,361 @@
...
+
+</TabItem>
+</Tabs>
 
 ## Prerequisites
 
-This chapter describes the prerequisites installation needed by plugins to run.
+The remote server must have an X509's certificate.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-protocol-x509.md:48]: ?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-x509.md [hunk 4]
```diff
@@ -2,59 +2,362 @@
...
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+The remote server must have an X509's certificate.
```
[the remark this answers is not in the document]
Comment (omercier) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-x509.md:49]: * Oups
* Le serveur http

[the remark this answers is not in the document]
Comment (omercier) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-x509.md:49]: "Le serveur HTTP doit supporter HTTPS et présenter un certificat X509." ?

File: pp/integrations/plugin-packs/procedures/applications-protocol-x509.md (hunk 1 above)
Comment (omercier) [reply pp/integrations/plugin-packs/procedures/applications-protocol-x509.md:41]: nope

File: pp/integrations/plugin-packs/procedures/applications-protocol-x509.md (hunk 3 above)
Comment (omercier) [reply pp/integrations/plugin-packs/procedures/applications-protocol-x509.md:48]:
```suggestion
The monitored resource must support HTTPS and present a X509 certificate.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-x509.md [hunk 5]
```diff
@@ -2,59 +2,362 @@
...
+Voici le tableau des services pour ce connecteur, détaillant les métriques rattachées à chaque service.
+
+<Tabs groupId="sync">
+<TabItem value="X509-Certificate" label="X509-Certificate">
+
+| Métrique    | Unité |
+|:------------|:------|
+| status      | N/A   |
+
+> Pour obtenir ce nouveau format de métrique, incluez la valeur **--use-new-perfdata** dans la macro de service **EXTRAOPTIONS**.
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-x509.md:42]:
```suggestion
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-x509.md [hunk 6]
```diff
@@ -2,59 +2,362 @@
...
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+The remote server must have an X509's certificate.
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-protocol-x509.md:49]:
```suggestion
L'équipement supervisé doit supporter HTTPS et présenter un certificat X509.
```

File: pp/integrations/plugin-packs/procedures/applications-protocol-x509.md [hunk 7]
```diff
@@ -2,59 +2,361 @@
...
+Here is the list of services for this connector, detailing all metrics linked to each service.
+
+<Tabs groupId="sync">
+<TabItem value="X509-Certificate" label="X509-Certificate">
+
+| Metric name | Unit  |
+|:------------|:------|
+| status      | N/A   |
+
+> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/applications-protocol-x509.md:41]:
```suggestion
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**